shell/platform/android/test/io/flutter/plugin/platform/PlatformPluginTest.java

```diff
@@ -85,15 +86,119 @@ public void platformPlugin_getClipboardData() throws IOException {
     PlatformPlugin platformPlugin = new PlatformPlugin(fakeActivity, fakePlatformChannel);
 
     ClipboardContentFormat clipboardFormat = ClipboardContentFormat.PLAIN_TEXT;
-    assertNull(platformPlugin.mPlatformMessageHandler.getClipboardData(clipboardFormat));
     ClipData clip = ClipData.newPlainText("label", "Text");
+
+    assertNull(platformPlugin.mPlatformMessageHandler.getClipboardData(clipboardFormat));
     clipboardManager.setPrimaryClip(clip);
     assertNotNull(platformPlugin.mPlatformMessageHandler.getClipboardData(clipboardFormat));
+  }
+
+  @Test
+  public void platformPlugin_getClipboardDataIsNonNullWhenContentUriWithTextProvided()
+      throws IOException {
```

Comment:
Can any of this setup logic be extracted into a logical helper method to share between tests?